docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -441,6 +441,42 @@ quarkus.kubernetes.readiness-probe.initial-delay=20s
 quarkus.kubernetes.readiness-probe.period=45s
 ----
 
+=== Add hostAliases
+To add entries to a Pod's `/etc/hosts` file (more information can be found in https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/[Kubernetes documentation]), just apply the following configuration:
+
+[source]
+----
+quarkus.kubernetes.host-aliases."10.0.0.0".hostnames=foo.com
+----
+
+This would generate the following section in the deployment descriptor:
+
+[source,yaml]
+----
+hostAliases:
```

Comment:
This is misleading, no?

It doesn't show the quarkus or the kubernetes group.
Furthermore, I am pretty sure it's `host-aliases`, right?

Reply:
Actually, the kubernetes directive is : **hostAliases**
I renamed quarkus config name to match it, it was indeed misleading here ;-)

Reply:
What I mean is that the Quarkus configuration would be 

```yaml
quarkus:
   kubernetes:
     host-alieases:
     ....
``` 


(as you have already speciified in the properties version)

Reply:
Ok I understand.
What I wanted to show here in the doc is actually an excerpt of the generated kubernetes.yaml file.
Is more clear like this : 
```suggestion
This would generate the following `hostAliases` section in the `deployment` definition:

[source,yaml]
----
kind: Deployment
spec:
  template:
    spec:
      hostAliases:
      - hostnames:
        - foo.com
        - bar.org
        ip: 10.0.0.0
----
```